content/cli/commands/table-format.md

```diff
@@ -0,0 +1,286 @@
+---
+title: Table Format 
+weight: 20
+---
+
+Table Format is used to customize the behavior of the Fluvio consumer output type [`full_table`]. 
+
+[`full_table`]: {{< ref "#full_table" >}}
+
+With `tableformat`, you can control the column labels, column ordering and control which keys are primary for displaying your live event data as row updates.
+
+```
+$ fluvio tableformat -h
+fluvio-tableformat 0.0.0
+Create a tableformat display specification
+
+USAGE:
+    fluvio tableformat <SUBCOMMAND>
+
+FLAGS:
+    -h, --help    Prints help information
+
+SUBCOMMANDS:
+    create    Create a new TableFormat display
+    delete    Delete a TableFormat display
+    list      List all TableFormat display
+    help      Prints this message or the help of the given subcommand(s)
+```
+
+This is the schema for the Table Format yaml config used by `fluvio tableformat create`
+
+You only need to give your Table Format a name, and an input format (currently only JSON is supported)
+
+### Table Format Config schema
```

Comment:
It would be good to get a step-by-step.

I copied the file and tried to run the command but I got an error.

% fluvio tableformat create --config format.yanl
Error: 
   0: Fluvio connector config
   1: name: invalid type: map, expected a string at line 5 column 9

Reply:
Maybe that was just a description, but I didn't quite read it that way.

Reply:
This is just a description. I've added some more words to make that more clear. And a link to the examples below, which is more of the copy/paste type of content with example inputs and a rendered output to compare